src/parser/jobs/mnk/MnkAoE.js

```diff
@@ -0,0 +1,80 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+
+import Module from 'parser/core/Module'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const ROCKBREAKER_MIN_TARGETS = 2
+const AOTD_MIN_TARGETS = 5
+
+export default class MnkAoE extends Module {
+	static handle = 'mnkaoe'
```

Comment:
Remind me to rename the AoE event generation module so actual aoe modules don't have to have silly prefixes